website/content/en/docs/deployment/notebook-culling-configuration.md

```diff
@@ -0,0 +1,35 @@
++++
+title = "Configure Culling for Notebooks"
+description = "Configure Notebook Culling Feature"
+weight = 70
++++
+
+### (Optional) 
+You can expose the timestamp of the last-activity of a Notebook instance and enable culling mechanism with that information. The notebook instance will be "culled" if it has been idled for the specified period of time. The design doc of the feature can be found in [here](https://github.com/kubeflow/kubeflow/blob/master/components/proposals/20220121-jupyter-notebook-idleness.md)
```

Comment:
The culling feature allows you to stop a Notebook Server based on its Last activity. The Notebook Controller updates the respective `notebooks.kubeflow.org/last-activity` annotation of each Notebook resource according to the execution state of the kernels. If this feature is enabled, the notebook instances will be "culled"(scaled to zero) if none of the kernels are performing computations for a specified period of time(CULL_IDLE_TIME). More information about this feature can be found in the [Jupyter notebook idleness proposal]

> Note: This feature only works with Jupyter notebooks